eng/scripts/docs/tests/Service-Readme-Generation-Tests.ps1

```diff
@@ -0,0 +1,44 @@
+<#
+```
+Invoke-Pester -Output Detailed $PSScriptRoot/Service-Readme-Generation-Tests.ps1
+```
+#>
+
+Import-Module Pester
+
+BeforeAll {
+    . $PSScriptRoot/../Docs-ToC.ps1
+    . $PSScriptRoot/../../../common/scripts/logging.ps1
```

Comment:
Should this not just include `common.ps1`? Wouldn't that also pull in `Docs-ToC.ps1`?

Reply:
We have common.ps1 import at common script, so the logging.ps1 does not include.

Reply:
Discussed offline. Agreed with the points that people use other functions in common.ps1 shouldn't break the tests, so import common.ps1 is better than Doc-Toc.ps1 + least necessary helper scripts.

Reply:
Changed to import common.ps1 only.

Reply:
Will add comments on what function in which file I am testing.